examples/lie_labs_api.py

```diff
@@ -0,0 +1,20 @@
+import torch
+
+import theseus.labs.lie as lie
+
+g = lie.rand(lie.SE3, 5)
+x = torch.rand(5, 6)
+
+z = g.retract(x)
+
+try:
+    y = g + x
+except RuntimeError as e:
+    print(e)
+
+y = g + lie.TangentTensor(x)
```

Comment:
Note that `TangentTensor` should be able to support arbitrary torch operations. It's a direct subclass of torch.Tensor, only with added ltype.